packages/client/edge/index.d.ts

```diff
@@ -1 +1 @@
-export * from '../.prisma/client'
+export * from '.prisma/client'
```

Comment:
The export was wrong, we're exporting the `.prisma/client` module, so not relative.